app/directives/proposal/proposal.directive.html

```diff
@@ -1,35 +1,40 @@
 <div class="prop-element">
 
     <div class="prop-row">
-        <div class="prop-row-child ng-class:[{'col-xs-8': (c.mode !== c.mds.COLLAPSED)},{'col-xs-10': (c.mode === c.mds.COLLAPSED)},{'col-sm-7': (c.mode !== c.mds.COLLAPSED)},{'col-sm-11': (c.mode === c.mds.COLLAPSED)},{'col-md-9': (c.mode !== c.mds.COLLAPSED)},{'col-md-11': (c.mode === c.mds.COLLAPSED)}]">
-            <p class="prop-text-header"><b>{{c.typeText(proposal.proposalType)}}</b>&nbsp;&nbsp;<span ng-if="!c.isUser">{{proposal.personName}}:</span>
-                <span ng-if="c.isUser" class="glyphicon glyphicon-user"></span>&nbsp;&nbsp;&nbsp;<span ng-if="(c.mode === c.mds.COLLAPSED)"
-                class="animate-if" ng-bind-html="proposal.text"></span></p>
-            <!--<span ng-if="c.isUser" class="glyphicon glyphicon-user"></span>-->
+        <div class="prop-row-child ng-class:[{'col-xs-8': (c.mode !== c.mds.COLLAPSED)},{'col-xs-7': (c.mode === c.mds.COLLAPSED && !c.isPbl)},{'col-xs-9': (c.mode === c.mds.COLLAPSED && c.isUser && c.isPbl)},{'col-xs-10': (c.mode === c.mds.COLLAPSED && !c.isUser && c.isPbl)},{'col-sm-7': (c.mode !== c.mds.COLLAPSED)},{'col-sm-8': (c.mode === c.mds.COLLAPSED && !c.isPbl)},{'col-sm-9': (c.mode === c.mds.COLLAPSED && c.isUser && c.isPbl)},{'col-sm-10': (c.mode === c.mds.COLLAPSED && !c.isUser && c.isPbl)},{'col-md-9': (c.mode !== c.mds.COLLAPSED)},{'col-md-10': (c.mode === c.mds.COLLAPSED)}]">
```

Comment:
Controller side declaration does not easily follow value changes (example collapsed or not) and therefore current implementation is acceptable. 
Layout finetuning for different browsers is still needed.
This can be merged if LGTM.
Branch is available on 0001121 page.